src/hardware/input/mouse_config.cpp

```diff
@@ -42,6 +42,10 @@ constexpr auto capture_type_onclick_str  = "onclick";
 constexpr auto capture_type_onstart_str  = "onstart";
 constexpr auto capture_type_nomouse_str  = "nomouse";
 
+constexpr auto model_dos_2button_str = "2button";
+constexpr auto model_dos_3button_str = "3button";
+constexpr auto model_dos_wheel_str   = "wheel";
+
```

Comment:
Constexpr is UpperCamelCase by convention, and another minor comment, I'm 50/50 on the `_str` prefix. Dangerously close to Hungarian notation, and we don't put `_str` after every string constant.

Reply:
Actually, I have already renamed these in the next commit - but OK, I have moved the cleanup here.

Reply:
Yeah, I generally review commit by commit and I usually go back to delete my redundant commens,, but forgot about this one.